src/js/util.js

```diff
@@ -125,6 +158,8 @@ exports.isObject = isObject;
 exports.isFunction = isFunction;
 exports.isBuffer = isBuffer;
 exports.isArray = Array.isArray;
+exports.exceptionWithHostPort = exceptionWithHostPort;
+exports.errnoException = errnoException;
```

Comment:
I'm not sure these functions are in utils.js or not.

util.js has compatibility with util in node.js.